com.lynnmichaelmartin.TimeTracker.json

```diff
@@ -0,0 +1,46 @@
+{
+    "id" : "com.lynnmichaelmartin.TimeTracker",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "46",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.node18",
+        "org.freedesktop.Sdk.Extension.typescript"
+    ],
+    "command" : "com.lynnmichaelmartin.TimeTracker",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home:create"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/typescript/bin:/usr/lib/sdk/node18/bin"
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "time-tracker",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "file:///home/lynn/Projects"
```

Comment:
what does that reference???

Reply:
Sorry, I'm new to app development. I guess that url should be https://github.com/elvishcraftsman/time-tracker.git

Reply:
> Sorry, I'm new to app development. I guess that url should be https://github.com/elvishcraftsman/time-tracker.git

Yes